NevisExampleApp/Domain/Use Case/DeregistrationUseCaseImpl.swift

```diff
@@ -41,7 +41,7 @@ extension DeregistrationUseCaseImpl: DeregistrationUseCase {
 		}
 
 		var responses: [Observable<OperationResponse>] = []
-		authenticators.forEach { authenticator in
+		for authenticator in authenticators {
```

Comment:
I think the SwiftFormat rule is missing here not to rewrite the forEach

Reply:
Fixed.